<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsThemeManagerWidgetBase</class>
 <widget class="QgsDockWidget" name="QgsThemeManagerWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>293</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Theme Manager</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,6,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,5,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="mThemePrev">
          <property name="toolTip">
           <string>Select previous theme</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../images/images.qrc">
            <normaloff>:/images/themes/default/mActionAtlasPrev.svg</normaloff>:/images/themes/default/mActionAtlasPrev.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="mThemeList">
          <property name="toolTip">
           <string>Select theme</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mThemeNext">
          <property name="toolTip">
           <string>Select next theme</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../images/images.qrc">
            <normaloff>:/images/themes/default/mActionAtlasNext.svg</normaloff>:/images/themes/default/mActionAtlasNext.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QgsThemeViewer" name="mThemeViewer" native="true">
        <property name="minimumSize">
         <size>
          <width>72</width>
          <height>0</height>
         </size>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="dragEnabled" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="horizontalSpacing">
         <number>2</number>
        </property>
        <item row="0" column="2">
         <widget class="QToolButton" name="mCreateTheme">
          <property name="toolTip">
           <string>Create New Theme</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../images/images.qrc">
            <normaloff>:/images/themes/default/mActionNewReport.svg</normaloff>:/images/themes/default/mActionNewReport.svg</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QToolButton" name="mAddThemeLayer">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Import Selected Layers</string>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
          <property name="icon">
           <iconset resource="../../images/images.qrc">
            <normaloff>:/images/themes/default/mActionSharingImport.svg</normaloff>:/images/themes/default/mActionSharingImport.svg</iconset>
          </property>
          <property name="popupMode">
           <enum>QToolButton::InstantPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonIconOnly</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QToolButton" name="mRemoveThemeLayer">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Remove selected layers</string>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
          <property name="icon">
           <iconset resource="../../images/images.qrc">
            <normaloff>:/images/themes/default/mIconSelectRemove.svg</normaloff>:/images/themes/default/mIconSelectRemove.svg</iconset>
          </property>
          <property name="popupMode">
           <enum>QToolButton::InstantPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonIconOnly</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QToolButton" name="mDeleteTheme">
          <property name="toolTip">
           <string>Delete Current Theme</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../images/images.qrc">
            <normaloff>:/images/themes/default/mActionDeleteSelected.svg</normaloff>:/images/themes/default/mActionDeleteSelected.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDockWidget</class>
   <extends>QDockWidget</extends>
   <header>qgsdockwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsThemeViewer</class>
   <extends>QWidget</extends>
   <header>qgsthemeviewer.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
